pkg/gui/files_panel.go

```diff
@@ -247,7 +247,7 @@ func (gui *Gui) handleStageAll(g *gocui.Gui, v *gocui.View) error {
 
 func (gui *Gui) handleIgnoreFile(g *gocui.Gui, v *gocui.View) error {
 	file := gui.getSelectedFile()
-	if file == nil {
+	if file == nil || file.Name == ".gitignore" {
```

Comment:
could we go:
```
if file == nil {
  return nil
}
if file.Name == ".gitignore" {
    return gui.createErrorPanel("Cannot ignore .gitignore")
}
```
That way users who for whatever reason want to ignore the gitignore at least they'll know lazygit is specifically disallowing it rather than bugging out.

Reply:
Thanks. That absolutely makes sense.